lib/property_test.ex

```diff
@@ -5,8 +5,18 @@ defmodule PropertyTest do
 
   # TODO: moduledoc (overview of ptesting, shrinking)
 
+  # QUESTION: Should we support property and automatically include Stream.Data
+  # and check all in there? Something like:
+  #
+  #     property "foo bar and baz" do
+  #       check all ... do
+  #         ...
+  #       end
+  #     end
```

Comment:
Mh yeah that might be nice because importing `Stream.Data` everywhere is kind of a lot of stuff. I am 👍 on this.